### ISSUE TITLE
Devices not responding after 0.15.5

### ISSUE THREAD

**Issue by Gachapen** [WebThingsIO/zigbee-adapter#280]
My devices are not responding after updating to 0.15.5. I also tried 0.15.6, but got the same result. I am using Conbee II with a Pi 4. No device shows the correct state, and they don't react to actions. My power socket devices and one Hue device is greyed out. There seem to be two errors in the log of special interest. The first is that messages from a thermostat are invalid. The other is that `parseSeCurrentSummDeliveredAttr` function is unknown. The complete log can be found here: https://gist.githubusercontent.com/Gachapen/c1ed0b868b43826d8d119f2760b5a276/raw/8d9b89777d32de039041ef18a653626655fd0e61/webthings.log

Below are a few excerpts.

```
2021-02-02 18:37:49.170 ERROR  : Invalid message received: {
  "messageType": 8192,
  "data": {
      ...
      "properties": {
        ...
        "heatTarget": {
          "name": "heatTarget",
          "value": 0,
          "visible": true,
          "title": "Heat Target",
          "type": "number",
          "@type": "TargetTemperatureProperty",
          "unit": "degree celsius",
          "minimum": null,
          "maximum": 0,
          "multipleOf": 0.5,
          "links": [],
          "profileId": 260,
          "endpoint": 1,
          "clusterId": 513,
          "attr": "occupiedHeatingSetpoint",
          "attrId": 18,
          "fireAndForget": true,
          "bindNeeded": false,
          "configReportNeeded": false,
          "initialReadNeeded": true
        },
        ...
      },
      ...
  }
}
2021-02-02 18:37:49.172 ERROR  : Validation error: [
  {
    "keyword": "type",
    "dataPath": ".data.device.properties['heatTarget'].minimum",
    "schemaPath": "#/properties/minimum/type",
    "params": {
      "type": "number"
    },
    "message": "should be number"
  }
]
```

```
2021-02-02 18:37:49.283 ERROR  : zigbee-adapter: Unknown function: parseSeCurrentSummDeliveredAttr
2021-02-02 18:37:49.284 ERROR  : zigbee-adapter: Error handling frame_object
2021-02-02 18:37:49.286 ERROR  : zigbee-adapter: Unknown function: parseSeCurrentSummDeliveredAttr
```

```
2021-02-02 18:37:54.757 ERROR  : zigbee-adapter: Unknown function: parseSeCurrentSummDeliveredAttr
2021-02-02 18:37:54.758 ERROR  : zigbee-adapter: handleExplicitRx: Caught an exception parsing ZDO frame
2021-02-02 18:37:54.759 ERROR  : zigbee-adapter: Unknown function: parseSeCurrentSummDeliveredAttr
```

```
2021-02-02 18:38:27.306 ERROR  : Failed to get property on: getProperty: device: zb-0015bc002f00705f not found.
2021-02-02 18:38:27.308 ERROR  : Failed to get property instantaneousPower: getProperty: device: zb-0015bc002f00705f not found.
2021-02-02 18:38:27.309 ERROR  : Failed to get property current: getProperty: device: zb-0015bc002f00705f not found.
2021-02-02 18:38:27.310 ERROR  : Failed to get property frequency: getProperty: device: zb-0015bc002f00705f not found.
2021-02-02 18:38:27.311 ERROR  : Failed to get property voltage: getProperty: device: zb-0015bc002f00705f not found.
2021-02-02 18:38:27.312 ERROR  : Failed to get property temperature: getProperty: device: zb-0015bc002f00705f not found.
2021-02-02 18:38:27.313 ERROR  : Failed to get property on: getProperty: device: zb-0015bc002f0071d0 not found.
2021-02-02 18:38:27.315 ERROR  : Failed to get property instantaneousPower: getProperty: device: zb-0015bc002f0071d0 not found.
2021-02-02 18:38:27.316 ERROR  : Failed to get property current: getProperty: device: zb-0015bc002f0071d0 not found.
2021-02-02 18:38:27.317 ERROR  : Failed to get property frequency: getProperty: device: zb-0015bc002f0071d0 not found.
2021-02-02 18:38:27.318 ERROR  : Failed to get property voltage: getProperty: device: zb-0015bc002f0071d0 not found.
2021-02-02 18:38:27.319 ERROR  : Failed to get property temperature: getProperty: device: zb-0015bc002f0071d0 not found.
2021-02-02 18:38:27.320 ERROR  : Failed to get property level: getProperty: device: zb-0017880108a05f36 not found.
2021-02-02 18:38:27.321 ERROR  : Failed to get property on: getProperty: device: zb-0017880108a05f36 not found.
```

```
2021-02-02 18:54:00.554 ERROR  : zigbee-adapter: No handler for ZDO cluster: 0000
2021-02-02 18:54:00.555 ERROR  : zigbee-adapter: { response: true,
2021-02-02 18:54:00.556 ERROR  : zigbee-adapter:   type: 23,
2021-02-02 18:54:00.557 ERROR  : zigbee-adapter:   seqNum: 120,
2021-02-02 18:54:00.558 ERROR  : zigbee-adapter:   status: 0,
2021-02-02 18:54:00.559 ERROR  : zigbee-adapter:   payloadLen: 34,
2021-02-02 18:54:00.559 ERROR  : zigbee-adapter:   deviceState: 34,
2021-02-02 18:54:00.560 ERROR  : zigbee-adapter:   networkState: 2,
2021-02-02 18:54:00.561 ERROR  : zigbee-adapter:   dataConfirm: false,
2021-02-02 18:54:00.562 ERROR  : zigbee-adapter:   dataIndication: false,
2021-02-02 18:54:00.563 ERROR  : zigbee-adapter:   configChanged: false,
2021-02-02 18:54:00.563 ERROR  : zigbee-adapter:   dataRequest: true,
2021-02-02 18:54:00.564 ERROR  : zigbee-adapter:   dstAddrMode: 2,
2021-02-02 18:54:00.565 ERROR  : zigbee-adapter:   destination16: 'fffc',
2021-02-02 18:54:00.566 ERROR  : zigbee-adapter:   destinationEndpoint: '00',
2021-02-02 18:54:00.566 ERROR  : zigbee-adapter:   srcAddrMode: 2,
2021-02-02 18:54:00.567 ERROR  : zigbee-adapter:   remote16: 'a6df',
2021-02-02 18:54:00.568 ERROR  : zigbee-adapter:   sourceEndpoint: '00',
2021-02-02 18:54:00.569 ERROR  : zigbee-adapter:   profileId: '0000',
2021-02-02 18:54:00.569 ERROR  : zigbee-adapter:   clusterId: '0000',
2021-02-02 18:54:00.570 ERROR  : zigbee-adapter:   data: <Buffer 60 87 e8 04 ff ff 2e 21 00 00 00>,
2021-02-02 18:54:00.571 ERROR  : zigbee-adapter:   lqi: 143,
2021-02-02 18:54:00.572 ERROR  : zigbee-adapter:   rssi: -76,
2021-02-02 18:54:00.572 ERROR  : zigbee-adapter:   zdoSeq: 96,
2021-02-02 18:54:00.573 ERROR  : zigbee-adapter:   addr64: '00212effff04e887',
2021-02-02 18:54:00.574 ERROR  : zigbee-adapter:   requestType: 0,
2021-02-02 18:54:00.575 ERROR  : zigbee-adapter:   startIndex: 0,
2021-02-02 18:54:00.575 ERROR  : zigbee-adapter:   remote64: '0017880106f70ad9' }
```

The last one is from a later log.

**Comment by mrstegeman** [comment]
I just released v0.15.7. It should fix the `parseSeCurrentSummDeliveredAttr` errors. I don't _believe_ the invalid message error should actually cause any issues.

**Comment by Gachapen** [comment]
Yes, everything is back to normal. Thank you!
